armory/data/datasets.py

```diff
@@ -550,6 +565,7 @@ def ucf101(
     batch_size: int = 1,
     dataset_dir: str = None,
     preprocessing_fn: Callable = None,
+    cache_dataset: bool = True,
 ) -> ArmoryDataGenerator:
     """
     UCF 101 Action Recognition Dataset
```

Comment:
Are UCF and Librispeech the only datasets we want to cache, or would other ones be helpful as well?

Reply:
resisc45 took me 10+ minutes to extract through TFDS, though the full dataset is only 390MB, which took me ~30 seconds to download, hash, and extract from .tar.gz.